static/app/views/performance/landing/utils.tsx

```diff
@@ -49,6 +50,13 @@ export const LANDING_DISPLAYS = [
     label: 'Backend',
     field: LandingDisplayField.BACKEND,
   },
+  {
+    label: 'Mobile',
+    field: LandingDisplayField.MOBILE,
+    isEnabled: (organization: Organization) =>
```

Comment:
I'd maybe rename this 'isShown' for clarity. I'd expect a UI element's 'enabled' to be css state usually (visible but grayed out).

Reply:
changed in d79cabf